gradle.properties

```diff
@@ -17,8 +17,8 @@
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
 
-VERSION_CODE=9
-VERSION_NAME=1.3.0-SNAPSHOT
+VERSION_CODE=10
+VERSION_NAME=1.3.0
```

Comment:
thats kinda weird IMO. Since I'd like to be able to checkout a sha and build it as if its the build we shipped.